tests/vcs/git/git_fixture.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+import shutil
+import typing
+
+
+if typing.TYPE_CHECKING:
+    from pathlib import Path
+
+    import dulwich.repo
+
+
+GIT_NOT_INSTALLLED = shutil.which("git") is None
```

Comment:
I think we can put this constant back into `test_system` because it is unlikely that it is relevant for other tests than for `SystemGit`.

Reply:
done